src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -69,33 +61,9 @@ private FlutterSdk(@NotNull final String homePath) {
     this(homePath, FlutterSdkUtil.getSdkVersion(homePath));
   }
 
-  /**
-   * Returns the same as {@link #getGlobalFlutterSdk()} but much faster
-   */
   @Nullable
   public static FlutterSdk getFlutterSdk(@NotNull final Project project) {
```

Comment:
It was done in the Dart plugin for 2 reasons:
- at that time reference resolution was done by IDE (now by server) and getGlobalDartSdk() was called thousands times a second
- bottleneck of getGlobalDartSdk() is java.io.File.exists(). It takes much less than 1ms, but when called thousands times a second it was visible in CPU snapshot

Neither is the case for Flutter. It is neither called frequently nor has File.exists() in its implementation.